### PR TITLE
Chief Engineer turtlenecks and garment changes.

### PR DESCRIPTION
**Pull request by oldyezero**
## About The Pull Request
Adds the Chief Engineer's turtleneck and turtleneck skirt to the game, and makes them and the Chief Medical Officer's turtlenecks available from round start.

fakes lolas

## Why It's Good For The Game
Provides more clothing variety for the Chief Engineer and makes the turtlenecks more accessible to the Chief Medical Officer.
## Changelog
:cl: myraowo
add: Added the Chief Engineer's turtleneck and turtleneck skirt to the game.
add: Added the Chief Engineer and Chief Medical Officer's turtlenecks to their respectives garment bags.

/:cl:

### CODE REVIEW

**Comment by jughu** [comment]
just make turtleneck a clothesvendor gag at this point lol

**Comment by optimumtact** [comment]
You're not allowed to add multiple uniforms to the job except for the formal outfits for staff.

These have to go in a clothing vendor.

**Review by Wallemations — CHANGES_REQUESTED** [review]
Even outside of the fact that these shouldn't be roundstart clothes, the color choice really doesn't work well for the visuals of the turtleneck. Maybe darker pants or a more orange top?

**Comment by Jackal-boop** [comment]
the CE wouldnt be caught dead wearing a dads tennis fit

**Comment by Admiral-Gigaton** [comment]
Ss14 sprites ass, fr

**Comment by SomeoneYouProbablyKnow** [comment]
Oh the turtleneck is not white. Shame. 

> You're not allowed to add multiple uniforms to the job except for the formal outfits for staff.
These have to go in a clothing vendor

Would making uniforms works same way as det revolver with a one time alt-click visual selection fix it? Or is the reason for this something other than locker bloat

**Comment by Admiral-Gigaton** [comment]
> Oh the turtleneck is not white. Shame.
> 
> > You're not allowed to add multiple uniforms to the job except for the formal outfits for staff.
> > These have to go in a clothing vendor
> 
> Would making uniforms works same way as det revolver with a one time alt-click visual selection fix it? Or is the reason for this something other than locker bloat

Clothing beacon pr incoming

**Comment by Higgin** [comment]
> the CE wouldnt be caught dead wearing a dads tennis fit

the CE is the head most likely to be caught dead wearing a dads tennis fit

that yellow is so garish i fucking love it

**Comment by oldyezero** [comment]
Are any of these two better?
1. Original
2. RD turtleneck recolor
2. RD turtleneck recolor (top only and fixed pants)


preview

**Comment by Ghommie** [comment]
> Are any of these two better?
> 
>     1. Original
> 
>     2. RD turtleneck recolor
> 
>     3. RD turtleneck recolor (top only and fixed pants)

I feel ambivalent between 2 and 3.

**Comment by Wallemations** [comment]
> Are any of these two better?
> 
> 1. Original
> 
> 2. RD turtleneck recolor
> 
> 2. RD turtleneck recolor (top only and fixed pants)
> 
> 
> 
> 
> 
> preview
> 
> 

Neither of these really fix the problem of the colors not working.
I hope you're not just hue-shifting a sprite and are actually using a palette

Either way I'm really not sure that a CE should be wearing a turtleneck. They're technicians not paper-pushers

**Comment by SomeoneYouProbablyKnow** [comment]
What about making the pants yellow and the turtleneck white with green highlights. O just don't think a solid piece of yellow is gonna work well (not much you can do about it being solid that how the turtleneck is)

**Comment by oldyezero** [comment]
> 
> Neither of these really fix the problem of the colors not working. I hope you're not just hue-shifting a sprite and are actually using a palette
> 
> Either way I'm really not sure that a CE should be wearing a turtleneck. They're technicians not paper-pushers

Yeah I used the first one's colors and copied the RD turtleneck sprite with them.

But if dirtiness is the issue, the HoS and CMO having turtlenecks wouldn't make much sense either, if they're all going to be covered in blood anyways



> What about making the pants yellow and the turtleneck white with green highlights. O just don't think a solid piece of yellow is gonna work well (not much you can do about it being solid that how the turtleneck is)

I honestly don't see any issue at all with it being to yellow, since every engineering clothing uses yellow as the base color, the CE jumpsuit/skirt being the different one and using white as another (not decal) color, much like this turtleneck.

Altough I will try to make the turtleneck match more with the actual jumpsuit colors, to see if it looks any better.

**Comment by oldyezero** [comment]
Well here it is, I used the 3rd sprite from the image I sent before because I think it's the one that looks better.

1. Original sprite
2. Recolored with the CE's jumpsuit color palette

preview 2

**Comment by SomeoneYouProbablyKnow** [comment]
Oh hell yea number 2 rocks

**Comment by oldyezero** [comment]
Honestly I will have to settle down with the original colors, using the CE's jumpsuit sprites doesn't make it look good or unique at all, but that's just my opinion too.

fake lolas 2

File: code/modules/cargo/packs/engineering.dm [hunk 1]
```diff
@@ -214,6 +214,14 @@
 	contains = list(/obj/item/circuitboard/computer/sat_control)
 	crate_name= "shield control board crate"
 
+/datum/supply_pack/engineering/ceturtlenecks
```
Comment (Ghommie) [inline code/modules/cargo/packs/engineering.dm:217]: But why? Can't this be added the garments bag for the chief engineer?

Comment (jughu) [reply code/modules/cargo/packs/engineering.dm:217]: Because oranges said so

**Pull request merged**